src/AppBundle/Resources/views/templates/blog.html.twig

```diff
@@ -11,8 +11,8 @@
             <div class="blog-item">
                 <div class="grid">
                     <div class="grid__item desk--five-tenths lap--one-half palm--one-whole">
-                        {% if article.image|length and article.image|first.thumbnails['600x300'] is defined %}
-                                {% set image = article.image|first %}
+                        {% if article.excerpt.images|length and article.excerpt.images|first.thumbnails['600x300'] is defined %}
```

Comment:
That's right, it's not possible at the moment. We do not have any content of the ArticleDocument here.